documentation/internal/Overview-of-Navigation-&-URL-structure.md

```diff
@@ -163,47 +163,41 @@ Example: [https://balsamiq.cloud/scs7giw/ponj59g/rD5AC](https://balsamiq.cloud/s
 
 ---
 
-## `/applications/[appId]?mode=assignment`
+## `/review/[appId]`
 
-### Application assignment mode
+Overview/Start page for reviews associated with specific Application. User will see different content depending on permissions. For Reviewers who have assigned content to review, they will have the option to "Start" review, which will create a Review in the database and re-direct to the specific review page.
 
-The “home page” of a specific application during assignment of reviewers.
+## `/review/[appId]/[reviewId]`
 
-The Supervisor -- with the proper permission to assign reviewers -- can do the assignment of reviewers to one (or more) section when the application is submitted to a new stage (the assignment changes per stage). After assigned the reviewers can review the specific sections.
-
-Example: [https://balsamiq.cloud/scs7giw/ponj59g/r3DC3](https://balsamiq.cloud/scs7giw/ponj59g/r3DC3)
+Specific Review page. Assigned Reviewer(s) can mark questions as "Conform" or "Non-conform" and add comments.
 
 ---
 
-## `/applications/[appId]?mode=consolidation`
-
-### Application consolidation mode
+## `/consolidate/[appId]`
```

Comment:
Similarly I think this should be replaced with `/applications/[appId]/consolidates`

Reply:
Plural here is tricky, because it would have to be `/consolidations`. In this case I think `/applications/[appId]/consolidation` (singular) is okay. Not that there can't be multiple consolidations, but to me it seems weird to draw attention to that for some reason.

Reply:
Sure. I agree

Reply:
similar to above i think both should be singular